core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -1238,12 +1238,8 @@ private void parseFromClause(GenericLexer lexer, QueryModel model, QueryModel ma
                             model.setSampleByTimezoneName(expectExpr(lexer));
```

Comment:
This piece does't seem to be consistent with sql diagram at https://questdb.io/docs/reference/sql/sample-by/
According to diagram _with offset_ is not allowed after _time zone 'X'_
```

Reply:
for the record: I discussed this with Bolek as I had noticed the same. The grammar in docs makes it seem that it's either `with offset` (x)or `time zone`, but the impl allowed both in one query. not only the impl allows it, but there is even a test for this behaviour. 

I think it's quite useful, as it allows you to select a specific timezone and then apply an offset on top of that. that might be useful, as offset gives you finer granularity. so I think the docs should be updated, not the impl. I created https://github.com/questdb/questdb.io/issues/1115 to track this.